File: TsinghuaNetUWP/ViewModels/NetStateSsidBox.vb
Imports TsinghuaNetUWP.Helper

Public NotInheritable Class NetStateSsidBox
    Public Sub New(ssid As String, value As NetState)
        Me.Ssid = ssid
        Me.Value = value
    End Sub

    Public Property Ssid As String
    Public Property Value As NetState
End Class

Module NetStateHelper
    Public ReadOnly NetStateNameList As New List(Of String) From {"不登录", "Net", "Auth4", "Auth6"}
End Module

File: TsinghuaNet.Uno/TsinghuaNet.Uno.Shared/ViewModels/NetStateSsidBox.cs
using System.Collections.Generic;
using TsinghuaNet.Helpers;

namespace TsinghuaNet.Uno.ViewModels
{
    sealed class NetStateSsidBox
    {
        public NetStateSsidBox(string ssid, NetState value)
        {
            Ssid = ssid;
            Value = value;
        }

        public string Ssid { get; set; }
        public NetState Value { get; set; }
    }

    static class NetStateHelper
    {
        public readonly static List<string> NetStateNameList = new List<string>() { "不登录", "Net", "Auth4", "Auth6" };
    }
}
